readme.md

```diff
@@ -55,7 +55,7 @@ If you aren't sure, run `git submodule update` after every git pull, merge or ch
 
 For reference, the following dependencies are included in Git submodules:
 
-* [comtypes](https://github.com/enthought/comtypes), version 1.1.3
+* [comtypes](https://github.com/enthought/comtypes), version 1.1.4

```

Comment:
Strictly spoken, tthis is not true any more. However functionally, this is equal to 1.1.4 minus the gen_path patch Python 2 doesn't benefit from anyway.